content/source/docs/enterprise/workspaces/repo-structure.html.md

```diff
@@ -9,50 +9,27 @@ sidebar_current: "docs-enterprise2-workspaces-repo-structure"
 Terraform Enterprise integrates with version control repositories to obtain
 configurations and trigger Terraform runs. Structuring the repo properly is
 important because it determines which files Terraform has access to when
-Terraform is executed within Terraform Enterprise.
-
+Terraform is executed within Terraform Enterprise, and when Terraform plans will run.
 
 ## Structuring Repos for Multiple Environments
 
-There are three ways to structure the Terraform code in your repository to manage
-multiple environments (dev, stage, prod).
+There are three ways to structure the Terraform code in your repository to manage multiple environments (such as dev, stage, prod).
 
-Depending on your organization's use of version control, one method for
-multi-environment management may be better than the other.
+Depending on your organization's use of version control, one method for multi-environment management may be better than another.
 
 ### Multiple Workspaces per Repo (Recommended)
 
-This is the overall recommended approach as it enables users to create a pipeline
-to promote changes through environments without additional overhead in version
-control. The model is to have one repo, such as `terraform-networking`, which
-is connected to multiple workspaces — `networking-prod`, `networking-stage`,
-`networking-uat`. Each workspace can be configured to connect to that VCS repo
-in the workspace settings. Additionally each workspace can have
-a unique set of variables to configure the differences per environment.
-
-To make an infrastructure change, a user can open a pull request on the
-`terraform-networking` repo, which triggers a plan in all three connected workspaces.
-The user can then merge the PR and apply it in one workspace at a time, first with
-`networking-uat`, then `networking-stage`, and finally `networking-prod`. Eventually
-Terraform Enterprise will have functionality to enforce the stages in this pipeline.
-
-This model does not work if there are major environmental differences. For example, if the
-`networking-prod` workspace has 10 more unique resources than the `networking-stage` workspace,
-they likely cannot share the same Terraform configuration and thus cannot share the same repo.
-If there are major structural differences between environments, one of the below approaches
-can work.
+Using a single repo attached to multiple workspaces is the overall recommended approach, as it enables the creation of a pipeline to promote changes through environments, without additional overhead in version control. When using this model, one repo, such as `terraform-networking`, is connected to multiple workspaces — `networking-prod`, `networking-stage`, `networking-dev`. While the repo connection is the same in each case, each workspace can have a unique set of variables to configure the differences per environment.
```

Comment:
maybe sub `is the over all recommended approach` with `is generally considered best practice" 

just ruminating here --  im not too attached to either wording

Reply:
I'm not quite sure either. The way I read the current phrase, it's saying "this is recommended in most cases, unless there are contraindications". "best practice" sounds stronger to me than that. But I also agree the "overall recommended approach" phrase is a bit strange.

Maybe something like "is the simplest best-practice approach"?

Reply:
👍 on your new phrase.